RELEASE-NOTES.md

```diff
@@ -104,24 +104,97 @@ buckets.default.allow_mult = true
 
 Note: this is defined in riak_kv.schema
 
-#### Backend configuration
+#### `riak_kv` configuration
 
-For Riak 2.0.5 or older, Old style backend configuration same as 1.5
-should be written in `riak_kv` section of `advanced.config`. See
+There are two style to configure Riak 2.0 behind Riak CS:
+
+1. Reuse `app.config` of Riak 1.4
+2. Copy proper items in old `app.config` to `advanced.config`
+
+In case of 1., `add_paths` should be changed to new Riak CS binaries
+installed by Riak CS 2.0 package, from
+`"/usr/lib/riak-cs/lib/riak_cs-1.5.4/ebin"` to
+`"/usr/lib/riak-cs/lib/riak_cs-2.0.0/ebin"`.  ```
```

Comment:
Yes, I knew that. For release build with specific tag, unless we're using rebar.config.lock for making release, that is the final name. See that directory name for 1.5.4. But after this gets merged, let's confirm this before shipping.